helm/sealed-secrets/values.yaml

```diff
@@ -363,3 +363,8 @@ metrics:
     ## @param metrics.dashboards.namespace Namespace where Grafana dashboard ConfigMap is deployed
     ##
     namespace: ""
+
```

Comment:
Please, could you move this section after the extraDeploy key.

Reply:
added